plugins/rikki/heroeslounge/components/upcomingmatches/default.htm

```diff
@@ -21,7 +21,15 @@
                         [{{match.teams[0].region.title}}]
                         {% endif %}
                     </span>
-                    <span title="{{match.division.title}}" class="hideOverflow">{{match.division.title}}</span>
+                    <span class="hideOverflow">
+                        {% if match.division.playoff is not null %}
+                        {{match.division.playoff.title}}
+                        {% elseif match.playoff is not null %}
+                        {{match.playoff.title}}
+                        {% else %}
+                        {{match.division.title}}
+                        {% endif %}
```

Comment:
This seems like logic we will want to reuse elsewhere so should it be added on the match model?

Reply:
I wasn't entirely sure, but I don't think it's warranted. From the top of my head I can't think of another place where we need to be so selective with displaying match info.

Reply:
I think I already did this somewhere actually, I just can't remember where

Reply:
There is something similar in ViewMatch, but it is not quite the same